test/test.js

```diff
@@ -0,0 +1,33 @@
+import assert from 'assert';
+import http from 'http';
+import { promises as fs } from 'fs';
+import Server from '../index.js';
+
+describe('HTTP /GET', () => {
+    it('Express server must return ./resorces/pages/index.html', async () => {
+        const server = new Server();
+
+        await server.run();
+
+        const mainPage = await fs.readFile('./resources/pages/index.html');
+
+        const response = await new Promise(resolve => {
+            http.get('http://localhost:1337', resolve);
+        });
+
+        let data = '';
+
+        response.on('data', chank => {
+            data += chank;
+        });
+
+        await new Promise(resolve => {
+            response.on('end', resolve);
+        });
+
+        assert.strictEqual(response.statusCode, 200);
+        assert.strictEqual(data.toString(), mainPage.toString());
+
+        await server.close();
```

Comment:
Use hooks in you tests to start/stop server (see in [mocha docs](https://mochajs.org/#hooks)).
Otherwise e.g. if asssertions fail server will not be stopped.